packages/core/root/core-root-browser-internal/src/core_system.ts

```diff
@@ -106,20 +109,24 @@ export class CoreSystem {
 
     this.rootDomElement = rootDomElement;
 
-    this.i18n = new I18nService();
+    const logLevel: LogLevelId = injectedMetadata.env.mode.dev ? 'all' : 'warn';
+    this.loggingSystem = new BrowserLoggingSystem({ logLevel });
```

Comment:
Using the `all` level for development and `warn` level for production.

Reply:
If I'm following correctly, then we still allow specifying the log level for browser-side logger instances but don't adhere to these levels when configuring logging in `kibana.yml`. 
Won't the fact that the browser-side logger can't be enabled in prod at all going to raise some eyebrows?